packages/core/cli.js

```diff
@@ -51,6 +51,14 @@ if (userWantsGeneralHelp) {
   displayGeneralHelp();
   process.exit(0);
 }
+//clean up inputString to run as `truffle help <cmd>` if `help`, `--help` is present with a <cmd>
```

Comment:
This could be improved to explain why the following code blocks are necessary. I stared at this for a minute, then read the code to understand the comment.